packages/query/src/helpers/identify-txs.ts

```diff
@@ -26,32 +37,61 @@ const hasRelevantIbcRelay = (
       return false;
     }
 
-    if (!action.action.value.rawAction?.is(MsgRecvPacket.typeName)) {
+    const rawAction = action.action.value.rawAction;
+    if (!rawAction) {
       return false;
     }
 
-    const recvPacket = new MsgRecvPacket();
-    const success = action.action.value.rawAction.unpackTo(recvPacket);
-    if (!success) {
-      throw new Error('Error while trying to unpack Any to MsgRecvPacket');
+    if (rawAction.is(MsgRecvPacket.typeName)) {
+      const recvPacket = new MsgRecvPacket();
+      rawAction.unpackTo(recvPacket);
```

Comment:
comment: https://github.com/penumbra-zone/web/pull/1810#discussion_r1776314026

Reply:
`unpackTo` does not throw. If it does not succeed, you can read the boolean it returns. However, I think this case is not possible given

```ts
if (rawAction.is(MsgRecvPacket.typeName)) {
```

Specifically identifies the Any before trying to unpack it